integration/cluster/cluster.go

```diff
@@ -228,12 +271,46 @@ func (c *Component) writeConfig() error {
 		return fmt.Errorf("error creating data path: %w", err)
 	}
 
+	if len(c.RemoteWriteUrls) > 0 {
+		c.rulesPath, err = os.MkdirTemp(c.cluster.sharedPath, "rules")
```

Comment:
Please clean up these temp files after the run

Reply:
done

Reply:
Can I ask you to please leave comments to the author to resolve? Otherwise it becomes difficult to track.
Where is the cleanup happening?

Reply:
oh sorry my bad!

Reply:
@aminesnow are we cleaning up the temp files/directories somewhere?

Reply:
@dannykopping hey, just back from PTO.
yes we do, in the `cleanup` function line 398

Reply:
Ah, I had to checkout the PR to understand the flow. Got it, thanks :+1: